PDF2IMG/index.html

```diff
@@ -20,7 +20,9 @@ <h1>Datalogics PDF2IMG Documentation</h1>
  <h2>PDF2IMG</h2>
  <p>A command-line utility and API that converts PDF files to a variety of image formats, including PNG, JPG, TIFF, BMP, and more. The product is built upon the Adobe PDF Library and uses Adobe technology for unrivaled color management during the PDF conversion process. While PDF2IMG can be run with as few as two command line arguments, many other optional command line arguments are provided so that you can create custom output. PDF2IMG is available on both Windows and Linux platforms, 32 and 64 bit.</p>
  <br>
- <p>PDF2IMG's .NET interface is now available as an x64 Windows NuGet package.</p>
+ <p>PDF2IMG's .NET interface is now available as an x64 Windows NuGet package. Please see the section marked <b>Appendix: API Calls, .NET Interface </b> in the API reference guide for more information on how to use our PDF2IMG NuGet package.
```

Comment:
Would it make sense to drop a link to the nuget package here?  i.e. https://www.nuget.org/packages/PDF2IMG.LM.NET#readme-body-tab

Reply:
Yes, it wouldn't hurt anything and it might help more people discover the package. Added!